jobs/mattermost/templates/config/initial-config.json.erb

```diff
@@ -134,7 +132,6 @@
         "AtRestEncryptKey": "<%= p("mattermost.SqlSettings.AtRestEncryptKey") %>",
         "QueryTimeout": <%= p("mattermost.SqlSettings.QueryTimeout") %>,
         "ConnMaxLifetimeMilliseconds": <%= p("mattermost.SqlSettings.ConnMaxLifetimeMilliseconds") %>,
-        "EnablePublicChannelsMaterialization": <%= p("mattermost.SqlSettings.EnablePublicChannelsMaterialization") %>
```

Comment:
In removing this line, you need to remove the comma from the tail of the preceding line. Hurray for JSON strictness :)

Reply:
_Siiiiiiigh_ do you know how long I started at my commit and looked for _exactly this kind of error_ and didn't find any? :-| I'll fix this in my next commit!